src/views/states/list/StatesList.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  NodeNetworkStateModelGroupVersionKind,
+  NodeNetworkStateModelRef,
+} from 'src/console-models';
+import NodeNetworkStateModel from 'src/console-models/NodeNetworkStateModel';
+import { useNMStateTranslation } from 'src/utils/hooks/useNMStateTranslation';
+
+import {
+  ListPageBody,
+  ListPageCreateDropdown,
+  ListPageFilter,
+  ListPageHeader,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { V1beta1NodeNetworkState } from '@types';
+import { getResourceUrl } from '@utils/helpers';
+
+import StateRow from './components/StateRow';
+import useStateColumns from './hooks/useStateColumns';
+import useStateFilters from './hooks/useStateFilters';
+
+const StatesList: React.FC = () => {
```

Comment:
I don't know if you're following the same conventions as the kubevirt-plugin repo. If you are, then `FC` should be imported by itself. If you're not, ignore this. =)